Jenkinsfile

```diff
@@ -40,7 +40,19 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-def daos_branch = "master"
+def doc_only_change() {
+    def rc = sh script: '''git diff-tree --no-commit-id --name-only HEAD |
+                               grep -q -v -e "^doc$"''',
```

Comment:
We might want to include .md files in this.  For instance, vos developer documentation is in src/vos/README.md

Reply:
Good idea.  That could be a future enhancement.  Right now we are only looking at PRs that only touch the doc/ directory, per [CORCI-843](https://jira.hpdd.intel.com/browse/CORCI-843).